### PR TITLE
Add travis file

### LINKED ISSUE #8: This repo has no tests
Not even a simple syntax test is done....

### PR DESCRIPTION
**Pull request by nickvergessen**
Fix #8

### CODE REVIEW

**Comment by nickvergessen** [comment]
@LukasReschke @schiessle

File: README.md [hunk 1]
```diff
@@ -0,0 +1,9 @@
+# Usage report / Survey client
+
+Sends anonymized data to Nextcloud to help us to improve Nextcloud. You
+always have full control over he content send to Nextcloud and can disable
```
Comment (LukasReschke) [inline README.md]: "the content"

Comment (LukasReschke) [inline README.md]: shouldn't that also be "sent"? Not sure.

**Comment by LukasReschke** [comment]
lgtm

File: README.md (hunk 1 above)
Comment (nickvergessen) [inline README.md]: Fixed, also in info.xml where I got this from

**Comment by schiessle** [comment]
:+1:

**Pull request merged**